corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,18 +34,22 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
 {% endblock %}
 
 {% block page_content %}
-    <table id="data-dictionary-table" class="table">
-        <thead>
-            <th>Case Property</th>
-            <th>Type</th>
-            <th>Description</th>
-        </thead>
-        <tbody data-bind="foreach: activeCaseTypeData">
-            <tr>
-                <td data-bind="text: name"></td>
-                <td data-bind="text: type"></td>
-                <td data-bind="text: description"></td>
-            </tr>
-        </tbody>
-    </table>
+    <div class="row">
+        <div class="col-md-10">
+            <table id="data-dictionary-table" class="table">
+                <thead>
+                    <th>{% trans "Case Property" %}</th>
+                    <th>{% trans "Type" %}</th>
```

Comment:
Is this table sortable? I think it might be easier to reason with if `type` is first on the list (the same way the [downloadable case summary](https://india.commcarehq.org/a/enikshay-test/apps/view/e6c88dc7cf8c742cbf397eff07d9a245/summary/case/download/?lang=en) has it. Also, assuming this is "Case Type" and not "Data Type". 
But I'm guessing this UI will probably go through some UX testing before getting fully released